Optional listname for push and pull functions

pascalfleury opened the pull request:
This makes scripting these calls easier, rather than going one level down and using more "internal" functions.

This is very useful when you want to bind this in a call from a timer to update the tests regularly (e.g. when refreshing the calendar view).

Commit:
Improved the README to give better bootstrap information.

Commit:
Cleanup the markup.

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Fix a typo.

Commit:
Avoid sending the secret around as a key, this is now an error on the Google API.

pascalfleury committed:
fixed merge conflicts

pascalfleury committed:
Merge branch 'JulienMasson-master'

pascalfleury committed:
Add list name as optional to avoid having a user prompt.

This is very useful when you want to bind this in a call from a timer
to update the tests regularly (e.g. when refreshing the calendar view).

pascalfleury committed:
Add list name as optional to avoid having a user prompt.

This is very useful when you want to bind this in a call from a timer
to update the tests regularly (e.g. when refreshing the calendar view).

pascalfleury committed:
Add list name as optional to avoid having a user prompt.

This is very useful when you want to bind this in a call from a timer
to update the tests regularly (e.g. when refreshing the calendar view).

pascalfleury committed:
Add list name as optional to avoid having a user prompt.

This is very useful when you want to bind this in a call from a timer
to update the tests regularly (e.g. when refreshing the calendar view).

pascalfleury committed:
Typo and additional info about the refresh token in docs.

JulienMasson commented on org-gtasks.el:
```diff
@@ -454,13 +456,15 @@
                          (append (list "ALL") collection)
                        collection))
-	 (target (completing-read "Push: " collection)))
+	 (target (if (eq listname nil)
```
can you please also replace `(eq listname nil)` by `(null listname)` here

JulienMasson replied:
can you please also replace `(eq listname nil)` by `(null listname)` here

JulienMasson replied:
can you please also replace `(eq listname nil)` by `(null listname)` here

JulienMasson replied:
can you please also replace `(eq listname nil)` by `(null listname)` here

JulienMasson commented on org-gtasks.el:
```diff
@@ -454,13 +456,15 @@
                        collection))
-	 (target (completing-read "Push: " collection)))
+	 (target (if (eq listname nil)
+                     (completing-read "Push: " collection)
```
can you please also replace `(eq listname nil)` by `(null listname)` here

Commit:
Merge branch 'master' of https://github.com/pascalfleury/org-gtasks into optional_listname_2

Commit:
simplify null detection

Commit:
Merge remote-tracking branch 'upstream/master' into optional_listname_2

Commit:
Merge branch 'optional_listname_2' of https://github.com/pascalfleury/org-gtasks into optional_listname_2

Commit:
# Ceci est la combinaison de 2 commits.
# Ceci est le premier message de validation :

Add list name as optional to avoid having a user prompt.

This is very useful when you want to bind this in a call from a timer
to update the tests regularly (e.g. when refreshing the calendar view).

# Ceci est le message de validation numéro 2 :

Commit:
Add list name as optional to avoid having a user prompt.

    This is very useful when you want to bind this in a call from a timer
    to update the tests regularly (e.g. when refreshing the calendar view).

pascalfleury commented:
somehow anything I do with git will end up with an even more messed up history...

JulienMasson commented:
your master is not aligned with mine, that's why when you create Pull-Request you have unwanted commits.

You should do something like this to align your master:
``` shell
$ git remote add -f upstream https://github.com/JulienMasson/org-gtasks.git
$ git push --force origin 'upstream/master:refs/heads/origin/master'
```

pascalfleury commented:
That was one reason. But you have not seen the mess I made in my git
history :-)
I went ahead and read most of the git book, as I was basically dealing with
remote repositories completely wrong.
Thanks for your patience !
--paf

On Mon, Jan 25, 2021 at 9:55 AM Julien Masson <notifications@github.com>
wrote:

> your master is not aligned with mine, that's why when you create
> Pull-Request you have unwanted commits.
>
> You should do something like this to align your master:
>
> $ git remote add -f upstream https://github.com/JulienMasson/org-gtasks.git
> $ git push --force origin 'upstream/master:refs/heads/origin/master'
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/JulienMasson/org-gtasks/pull/9#issuecomment-766658321>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBP5KB3FRDBXR3ANJNIVXDS3UWVLANCNFSM4WNSV6QQ>
> .
>

JulienMasson commented:
> But you have not seen the mess I made in my git history :-)

ahahaha indeed

> I went ahead and read most of the git book, as I was basically dealing with
remote repositories completely wrong.
Thanks for your patience !

no problem :)
